index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <header id="header">
+        <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png" alt="logo" id="header-img" />
+        <nav id="nav-bar">
+<a href='#form' id="nav nav-link">form</a>
+    <a href='#video' id="nav nav-link">video </a>
+    <a href='#' id="nav nav-link">link3</a>
+        </nav>
+    </header>
+    <form id="form" action="https://www.freecodecamp.com/email-submit">
+        <input type="email" placeholder="enter your email" required/>
+        <button id="submit">Submit</button>
+
+    </form>
+    <div id="video">
```

Comment:
- **Optional**

- [ ] Very good👍 way to use id attributes, but for your code to be shorter and well structured it would be better to use class attributes to apply the style to several blocks or elements in the same page.